test/runtests.jl

```diff
@@ -2011,3 +2011,12 @@ end
     @test_throws ArgumentError lm(@formula(OptDen ~ Carb), form; method=:pr)
     @test_throws ArgumentError glm(@formula(OptDen ~ Carb), form, Normal(); method=:pr)
 end
+
+@testset "[G]VIF" begin
+    duncan = RDatasets.dataset("car", "Duncan")
+    lm1 = lm(@formula(Prestige ~ 1 + Income + Education), duncan)
+    @test termnames(lm1)[2] == coefnames(lm1)
+    @test vif(lm1) ≈ gvif(lm1)
+    lm2 = lm(@formula(Prestige ~ 1 + Income + Education + Type), duncan)
+    @test gvif(lm2; scale=true) ≈ [1.486330, 2.301648, 1.502666] atol=1e-4
```

Comment:
1. Can you please add a comment on where these values are taken from?
2. Also do we have tests for `vif`/`gvif` for `glm`?
3. Do we have tests for `vif`/`gvif` for models without formula?
4. Do we have tests for `vif`/`gvif` for models that have complex formulas, something like e.g `@formula(y~(1+a*(b+log(c)))&(1+d))`? (of course this is artificial, but I hope it is clear what I mean

Reply:
These are just the StatsModels tests carried forward to models actually fitted here. :smile: But I can add a cross reference.